candle-examples/examples/llama/main.rs

```diff
@@ -96,8 +96,8 @@ struct Args {
     seed: u64,
 
     /// The length of the sample to generate (in tokens).
-    #[arg(long, default_value_t = 100)]
-    sample_len: usize,
+    #[arg(long, default_value_t = 4096)]
```

Comment:
Could we keep the old argument name and default value? These were pretty helpful when benchmarking changes.